frontend/src/views/Auction/AuctionView.tsx

```diff
@@ -0,0 +1,82 @@
+import style from "./auction.module.scss";
+import { AuctionItem } from "interfaces/index";
+import { GreenETHValue, CreatorsListItem, ProfilePicture, RoundButton, Button } from "components";
+import shareImg from "assets/share.svg";
+import moreOptionsImg from "assets/threeDots.svg";
+import heartImg from "assets/heartRed.svg";
+
+interface AuctionViewProps {
+    auctionData: AuctionItem | undefined;
```

Comment:
Having this as optional key will do the same as defining it can be of type undefined

Reply:
And if you mean that it can be empty before fetching value, then use null